docs/flashing.md

```diff
@@ -78,6 +78,10 @@ or
 
     make <keyboard>:<keymap>:avrdude
 
+or if you want to flash multiple boards, please do as follows.
```

Comment:
`or if you want to flash multiple boards, use the following command`  may be a better way to word this.

Also, it may be a good idea to mention that you probably have to manually break the loop here. For instance:
"When you're done flashing boards, you'll need to hit `Ctrl + C` or whatever the correct keystroke is for your operating system to break the loop")

Reply:
Thank you for your advice.